src/oge/validation.py

```diff
@@ -250,14 +315,33 @@ def check_for_orphaned_cc_part_in_subplant(subplant_crosswalk):
         logger.warning(
             f"There are {len(orphaned_cc_parts)} subplants that only contain one part of a combined cycle system.\nSubplants that represent combined cycle generation should contain both CA and CT parts."
         )
-        logger.warning("\n" + orphaned_cc_parts.to_string())
+        id2ba = (
+            create_plant_ba_table(year).set_index("plant_id_eia")["ba_code"].to_dict()
+        )
+        logger.warning(
+            "\n"
+            + orphaned_cc_parts.assign(
+                ba_code=[
+                    id2ba[i]
+                    for i in orphaned_cc_parts.index.get_level_values("plant_id_eia")
+                ]
+            )
+            # merge(
```

Comment:
was this meant to be converted to merge? Otherwise why are we using a different format here?

Reply:
It is a multi-index data frame and the merge operation strip the second index (subplant_id). The current operation conserve the multi indexing.